gamemodes/terrortown/gamemode/client/cl_player_ext.lua

```diff
@@ -114,6 +114,18 @@ function plymeta:AnimUpdateGesture()
     end
 end
 
+---
+-- Creates an elliptic nick for a given length.
+-- @param number width The maximum width that should be used to limit the nick
+-- @param[default="DefaultBold"] string font The font ID
+-- @param[default="..."] string limitChar The limiting character(s) that might be appended to the end
+-- @param[default=1.0] number scale The UI scale factor
+-- @return string The length limited nick
+-- @realm client
+function plymeta:NickElliptic(width, font, limitChar, scale)
+    return draw.GetLimitedLengthText(self:Nick(), width, font, limitChar or "...", scale)
```

Comment:
Do we really want the `limitChar` to be configurable here? i think just using '...' here would be fine and is a bit more in line with the function name

Reply:
I don't care. If you say so, I'll change it